src/hoc.js

```diff
@@ -0,0 +1,3 @@
+import {ProvidedHOC} from '@uber/graphene-react';
```

Comment:
This is not correct

Reply:
I don't think know what this file does. Removing.